src/tree/SubscriptionTreeItem.ts

```diff
@@ -27,14 +31,32 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean): Promise<AzExtTreeItem[]> {
+
+        let treeItemPostgres: AzExtTreeItem[];
+        let treeItem: AzExtTreeItem[];
+
+        //Postgres
+        const clientPostgres: PostgreSQLManagementClient = createAzureClient(this.root, PostgreSQLManagementClient);
+        const accountsPostgres: ServerListResult = await clientPostgres.servers.list();
+        const databasesPostgres = await clientPostgres.databases.listByServer("myResourceGroup", accountsPostgres[0].name);
+        treeItemPostgres = await this.createTreeItemsWithErrorHandling(
+            accountsPostgres,
+            'invalidPostgreSQLAccount',
+            async (server: Server) => await this.initChild(clientPostgres, server, databasesPostgres),
+            (server: Server) => server.name
+        );
+
         const client: CosmosDBManagementClient = createAzureClient(this.root, CosmosDBManagementClient);
         const accounts: DatabaseAccountsListResult = await client.databaseAccounts.list();
-        return await this.createTreeItemsWithErrorHandling(
+        treeItem = await this.createTreeItemsWithErrorHandling(
             accounts,
             'invalidCosmosDBAccount',
             async (db: DatabaseAccount) => await this.initChild(client, db),
             (db: DatabaseAccount) => db.name
         );
+
+        treeItem.push(treeItemPostgres.pop());
```

Comment:
This would only add the first tree item. Something like this will add all items: `treeItems.push(...treeItemPostgres)`